ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-device-documentation.j2

```diff
@@ -9,6 +9,8 @@
 {% include 'documentation/cvx.j2' %}
 {# Authentication #}
 {% include 'documentation/authentication.j2' %}
+{# radius server #}
```

Comment:
Removed, because we using link to documentation/radius-servers.j2 in authntication.j2 and I modified the template documentation/radius-servers.j2 as requested below.